spec/models/user_spec.rb

```diff
@@ -0,0 +1,74 @@
+require 'rails_helper'
+
+RSpec.describe User, type: :model do
+  before do
+    User.destroy_all
```

Comment:
Why do we need to destroy all? rspec should cleanup automatically.

Reply:
Bhai, done.